lib/utils.pm

```diff
@@ -213,7 +213,7 @@ sub fully_patch_system {
 sub minimal_patch_system {
     my (%args) = @_;
     $args{version_variable} //= 'VERSION';
-    if (sle_version_at_least('12-SP1', version_variable => $args{version_variable})) {
+    if (sle_version_at_least('12-SP1', version_variable => $args{version_variable}) && !check_var('HDDVERSION', 'SLES-11')) {
```

Comment:
I don't think this is a good idea. It already tests for sle_version_at_least with a forward of the type of variable so it should do exactly what you want to do already